apps/condo/package.json

```diff
@@ -50,6 +50,7 @@
     "classnames": "^2.2.6",
     "date-fns": "^2.20.1",
     "dd-trace": "^0.32.0",
+    "diff": "^5.0.0",
```

Comment:
Please check the license, is BSD-3 ok for us? 
https://github.com/kpdecker/jsdiff/blob/master/LICENSE